src/web-worker/amp-worker.js

```diff
@@ -96,7 +96,7 @@ class AmpWorker {
     };
 
     if (self.trustedTypes && self.trustedTypes.createPolicy) {
-      const policy = self.trustedTypes.createPolicy(
+      policy = self.trustedTypes.createPolicy(
```

Comment:
I think it might just be better to rename the definition on line 78 to something like tempPolicy and switch out line 101 for the new name just to be completely sure we don't have any namespace conflicts

Reply:
sounds good

Reply:
went ahead and just switch it to a `let` to keep it consistent with the other code